cedar-policy-core/src/ast/extension.rs

```diff
@@ -355,7 +356,7 @@ pub struct ExtensionValueWithArgs {
 
 impl ExtensionValueWithArgs {
     /// Get the internal value
-    pub fn value(&self) -> &dyn InternalExtensionValue {
+    pub fn value(&self) -> &(dyn InternalExtensionValue) {
```

Comment:
Curious what this change does / what this syntax means?

Reply:
Ah, it's no-op. It was needed as I working through it as my first draft was changing `&dyn ...` to `&(dyn ... + Send + Sync`. Ended up pushing the bounds on `Send`/`Sync` further down